### DISCUSSION TITLE
DAI 2024-2025 - Practical work 2 - Thirusan Rajadurai and Thomas Stäheli

### DISCUSSION THREAD

**Discussion by Thiru-sama** [heig-vd-dai-course/heig-vd-dai-course#464] [Show and tell]
Our project idea is to make a connect 4 with a multiplayer (1v1).

**Comment by GeraudSilvestri** [comment]
The concept is good for me

Remember to start simple and add more complex features once the base is working as intended.

**Comment by thomasstaheli** [comment]
Good evening,

As asked, this is the link to our repository : [Repository connect4](https://github.com/thomasstaheli/DAI_Practical2_Connect4)
And this is the commit hash for the evaluation : 224469ca5901fc597a7d5fffcc71ae2cc02babc9 [Link to hash](https://github.com/thomasstaheli/DAI_Practical2_Connect4/commit/224469ca5901fc597a7d5fffcc71ae2cc02babc9)

See you tomorow ! 👍
